RegenerateServiceKey/handler.ts

```diff
@@ -55,7 +56,8 @@ type ResponseTypes =
   | IResponseErrorForbiddenNotAuthorized
   | IResponseErrorNotFound
   | IResponseErrorTooManyRequests
-  | IResponseErrorInternal;
+  | IResponseErrorInternal
+  | IResponseErrorValidation;
```

Comment:
Ditto. Btw Do we need this in `RegenerateServiceKey` ?

Reply:
`ResponseErrorValidation` is added to `ErrorResponses` of _utilis/responces.ts_,
 so it is added `IResponseErrorValidation` here too, to mach:

- `  .fold<ResponseTypes>(identity, identity)`

in handlers, but it is not correct because the other handlers may not return 400, thanks 💪